src/program/packetblaster/lwaftr/lib.lua

```diff
@@ -367,37 +368,45 @@ function Lwaftrgen:pull ()
       ipv4_udp_hdr.dst_port = C.htons(self.current_port)
       ipv6_ipv4_udp_hdr.src_port = C.htons(self.current_port)
 
-      for _,size in ipairs(self.sizes) do
+      -- The sizes are frame sizes, including the 4-byte ethernet CRC
+      -- that we don't see in Snabb.
+
+      local vlan_size = self.vlan and ether_vlan_header_size or 0
+      local ethernet_crc_size = 4
```

Comment:
More of an aesthetic nit but I think this would be better defined at the top as a "constant". WDYT?